jarbas/core/models.py

```diff
@@ -116,7 +116,11 @@ def as_list(content, cast=None):
             return None
 
         parts = list(content.split(','))
-        return list(map(lambda x: cast(x), parts)) if cast else parts
+        try:
+            return list(map(lambda x: cast(x), parts)) if cast else parts
+        except ValueError:
+            # this is far from good
+            return list()
```

Comment:
I'm assuming the `except ValueError` might happen in `cast`, right? If so, I don't think we should silence this error. If the user is trying to convert something unconvertible it's better to stop temh than to loose data silently.